### ISSUE TITLE
`@continue` blade directive has no highlight

### ISSUE THREAD

**Issue by cesarreyes3** [haidubogdan/netbeans-php-blade-plugin#67]
```blade
@foreach($tests as $test)  
  @continue(!$items->count() ? true : false)
@endforeach
```
![image](https://github.com/user-attachments/assets/df150616-b13b-4a3c-a5ec-06e3cae3938a)

**Comment by haidubogdan** [comment]
Hi,
Thank you for your observation.
A issue linked with directives which can also be without arguments. 
`@break` also had this issue.

Should be fixed with the latest prerelease 2.5.0.2 -  https://github.com/haidubogdan/netbeans-php-blade-plugin/releases/tag/prenbpr1.0.9

**Comment by cesarreyes3** [comment]
Another question, it is something minimal, if it complicates everything it would be better to ignore it,

On custom directives, if you click/select an opening directive, closing directive gets highlighted
![image](https://github.com/user-attachments/assets/e0ce6a35-9e27-4ace-89b2-f58c1d7f688d)

But if you click/select a closing directive, opening directive does nothing
![image](https://github.com/user-attachments/assets/3846bfd7-3792-43b0-9c5b-a4829d35ac5a)

In large blocks it's difficult to find the opening directive from closing directive
